### DISCUSSION TITLE
Can't use an iframe when using development server

### DISCUSSION THREAD

**Discussion by atrauzzi** [parcel-bundler/parcel#5995] [Q&A] [answered]
I get the message `Refused to display 'https://[whatever]/' in a frame because it set 'X-Frame-Options' to 'sameorigin'.` when trying to do a simple iframe with the development server.

```html
<!doctype html>
<html>
    <head>
        <title>No iframes!</title>
    </head>
    <body>
        <div>
            <iframe src="https://www.google.com"></iframe>
        </div>
    </body>
</html>
```

Is there any way to configure the `x-iframe-options` that it serves?

**Comment by DeMoorJasper** [comment]
Not sure where this error originates from but you might give proxy a try: https://v2.parceljs.org/features/api-proxy/

**Comment by mischnic** [comment, marked as answer]
`google.com` has the response header `x-frame-options: SAMEORIGIN`, not `localhost:1234`. So there's nothing you can do
